bot.py

```diff
@@ -124,21 +125,25 @@ async def on_message_delete(message):
         del starboard_messages[message.id]
 
 
+def is_trivial_command(c: str) -> Optional[str]:
+    cmd = c.strip()
+    if not cmd.startswith("?"):
+        return None
+    if cmd.count(" "):
+        return None
+
+    return cmds.get(cmd, None)
```

Comment:
is this case sensitive? if so, can it be insensitive?

Reply:
It can be case-insensitive by a simple `lower()` call but the question is do we want it?

Reply:
imagine the simple case of ?hello, someone may expect ?Hello to work. at least for these trivial commands it's probably good to add the lower()